converter.go

```diff
@@ -0,0 +1,308 @@
+	ignoreChars string
+}
+
+// MakeStringToUIntConverter creates StringToUIntConverter.
+func MakeStringToUIntConverter(ignoreChars string) StringToUIntConverter {
+	return StringToUIntConverter{ignoreChars: ignoreChars}
+}
+
+// Convert is the implementation of Converter[string, any] for StringToUIntConverter.
+func (conv StringToUIntConverter) Convert(src string) (any, error) {
+	src = replaceSeparators(src, conv.ignoreChars, "")
+	return strconv.ParseUint(src, 10, 64)
+}
+
+// StringToIntConverter is a converter from string to int64.
+type StringToIntConverter struct {
+	ignoreChars string
+}
+
+// MakeStringToIntConverter creates StringToIntConverter.
+func MakeStringToIntConverter(ignoreChars string) StringToIntConverter {
+	return StringToIntConverter{ignoreChars: ignoreChars}
+}
+
+// Convert is the implementation of Converter[string, any] for StringToIntConverter.
+func (conv StringToIntConverter) Convert(src string) (any, error) {
+	src = replaceSeparators(src, conv.ignoreChars, "")
+	return strconv.ParseInt(src, 10, 64)
+}
+
+// StringToFloatConverter is a converter from string to float64.
+type StringToFloatConverter struct {
+	ignoreChars   string
+	decSeparators string
+}
+
+// MakeStringToFloatConverter creates StringToFloatConverter.
+func MakeStringToFloatConverter(ignoreChars, decSeparators string) StringToFloatConverter {
+	return StringToFloatConverter{ignoreChars: ignoreChars, decSeparators: decSeparators}
+}
+
+// Convert is the implementation of Converter[string, any] for StringToFloatConverter.
+func (conv StringToFloatConverter) Convert(src string) (any, error) {
+	src = replaceSeparators(src, conv.ignoreChars, conv.decSeparators)
+	return strconv.ParseFloat(src, 64)
+}
+
+// StringToDecimalConverter is a converter from string to decimal.Decimal.
+type StringToDecimalConverter struct {
+	ignoreChars   string
+	decSeparators string
+}
+
+// MakeStringToDecimalConverter creates StringToDecimalConverter.
+func MakeStringToDecimalConverter(ignoreChars, decSeparators string) StringToDecimalConverter {
+	return StringToDecimalConverter{ignoreChars: ignoreChars, decSeparators: decSeparators}
+}
+
+// Convert is the implementation of Converter[string, any] for StringToDecimalConverter.
+func (conv StringToDecimalConverter) Convert(src string) (any, error) {
+	src = replaceSeparators(src, conv.ignoreChars, conv.decSeparators)
+	return decimal.NewDecimalFromString(src)
+}
+
+// StringToUUIDConverter is a converter from string to UUID.
+type StringToUUIDConverter struct{}
+
+// MakeStringToUUIDConverter creates StringToUUIDConverter.
+func MakeStringToUUIDConverter() StringToUUIDConverter {
+	return StringToUUIDConverter{}
+}
+
+// Convert is the implementation of Converter[string, any] for StringToUUIDConverter.
+func (StringToUUIDConverter) Convert(src string) (any, error) {
+	return uuid.Parse(src)
+}
+
+// StringToDateTimeConverter is a converter from string to datetime.Datetime.
+type StringToDateTimeConverter struct{}
```

Comment:
https://github.com/tarantool/tarantool/wiki/Datetime-Internals
```suggestion
// StringToDatetimeConverter is a converter from string to datetime.Datetime.
type StringToDatetimeConverter struct{}
```

Reply:
Changed.